deviceComunicationDocumentation.MD

```diff
@@ -179,38 +162,34 @@ Příklad:
 | 256          | 255                                                  |
 | 257          | 00  (došlo k přetečení a counter se restartoval)     |
 
-### 1.byte - identifikátor přijatých zpráv
+### 1.byte - počítadlo přijatých zpráv
 
-Defautlně je nastaveno na 0. Po přijetí zprávy ze serveru se nastaví na hodnotu
-přijatou ve zprávě ze serveru. Více o této hodnotě zde (TODO odkaz).
+Defautlně je nastaveno na 0. Po přijetí zprávy ze serveru se nastaví na hodnotu přijatou ve zprávě ze serveru. Více o této hodnotě zde (TODO odkaz).
 
 ### 2.byte - napětí baterie v zařízení
 
 Určuje hodnotu napětí baterie. Pokud je zařízení napájeno ze zásuvky tak obsahuje hodnotu napětí zdroje.
 
-Napětí může nabívat hodnot 1.8V–4.35V. Hodnota 0 odpovídá napětí 1.8V a hodnota 255 odpobvídá napětí 4.35V.
+Napětí může nabívat hodnot 1.8 V–4.35 V. Hodnota 0 odpovídá napětí 1.8 V a hodnota 255 odpovídá napětí 4.35 V.
 
-Aktuální napětí můžeme vypočítat podle následujícího vzorce: *naměřená hodnota* / 100 + 1.8
+Aktuální napětí můžeme vypočítat podle následujícího vzorce: *přijatá hodnota ve zprávě* / 100 + 1.8
 
 ### 3.byte - teplota procesoru
 
-Určuje teplotu procesoru. Tento byte může nabývat hodnotu 0-160 a 255. Aktuální
-teplotu vypočítáme podle následujícího vzorce: *přijatá hodnota* - 40 = aktuální teplota.
-Pokud je teplota vyšší než 120°C tak senzor pošle hodnotu 0xFF (255). Zařízení tedy
-dokáže měřit teploty -40°C až 120°C.
+Určuje teplotu procesoru. Tento byte může nabývat hodnotu 0-160 a 255. Aktuální teplotu vypočítáme podle následujícího vzorce: *aktuální teplota* = *přijatá hodnota ve zprávě* - 40.
+Pokud je teplota vyšší než 120°C tak senzor pošle hodnotu 0xFF (255). Celkový rozsah měřené teploty je -40 °C až 120 °C.
 
-Zařízení nikdy nepošle hodnotu 161 až 254. Jelikož by odpovídala hodnotě vyšší než 120°C.
+Zařízení nikdy nepošle hodnotu 161 až 254, která by odpovídala teplotám vyšším než 120 °C.
 
 ### 4.byte - RSSI
 
-Určuje RSSI (TODO odkaz na wiki) zařízení při odesílání zprávy.
+Obsahuje hodnotu RSSI (TODO odkaz na wiki) naměřeného při odesílání předchozí zprávy.
 
 ### 5.byte - potvrzení, disconnect a počet pokusů o odeslání zprávy
 
-Tento byte obsahuje několik informací o právě odesílané zprávě. Obsah byte se liší podle toho jestli
-zařízení používá síť NbIot nebo Lora.
+Tento byte obsahuje více informací o odesílané zprávě. Obsah bytu se liší podle toho, jestli zařízení používá síť NB-IoT nebo LoRa.
```

Comment:
já bych rekl ze "několik" je lepsi nez "více".   Takhle to zni jako ze tenhle byte obsahuje nějaké informace navíc.

Možná se mi to ale jen zdá.

Reply:
Nakonec jsem to tam nechal, mě to přijde, že to je stejné. Kdyžtak upravíme.